salesforce/10.10/modules/ROOT/pages/salesforce-connector-config-topics.adoc

```diff
@@ -1,12 +1,14 @@
-= Salesforce Connector 10.10 Additional Configuration - Mule 4
+= Salesforce Connector 10.10 Additional Configuration Information - Mule 4
 :page-aliases: connectors::salesforce/salesforce-connector-config-topics.adoc
 
-Configure these Anypoint Connector for Salesforce (Salesforce Connector) operations to:
+*Kim, I moved the last section, Generate an OAuth 2.0 Keystore file, from the Studio topic to this topic. It has more to do with Mule than the connector (but has some relation to the connnector), so the current intro does not work. I am not sure how to introduce these sections.*
+
+Configure Anypoint Connector for Salesforce (Salesforce Connector) to:
 
 * Load data in batches
 * Specify a Lead ID for a Lead Convert Request operation
-* Use the Replay topic and Replay channel listener operations
-* Configure a redelivery policy
+* Update Field Values to Null
```

Comment:
```suggestion
* Update field values to `null`
```
These are plain-text task list items, not references to the titles (per previous peer list items).